src/routes/app.js

```diff
@@ -62,7 +62,7 @@ function AppInner(props) {
   }
 
   return (
-    <>
+    <Fragment>
```

Comment:
y u like Fragment dawg

Reply:
I'd tell you something like it's more performant or easier for React to optimizing, but the truth is my editor bugs out at `<>` fragments, so I just change em to `<Fragment>`.

Reply:
haha - weird! we use the same editor. I will fix u someday